decision/engine.go

```diff
@@ -78,6 +81,9 @@ const (
 	// long/short term scores for tagging peers
 	longTermScore  = 10 // this is a high tag but it grows _very_ slowly.
 	shortTermScore = 10 // this is a high tag but it'll go away quickly if we aren't using the peer.
+
+	// Number of concurrent workers that process requests to the blockstore
+	blockstoreWorkerCount = 128
```

Comment:
Hm. This many _static_ workers is a bit annoying. How tricky would a dynamic worker pool be? This should only cost us 0.25MiB but that's at the _default_ stack size.

Note: this _isn't_ high priority. I'm fine merging as-is and testing in production.

Reply:
Probably it's not too hard to make it dynamic - I'd suggest we try it with a static worker pool size in production and see if it makes a difference, if so we can look at making it dynamic